admin/gv_mail.php

```diff
@@ -4,421 +4,382 @@
-<?php } ?>
-              <tr>
-                <td valign="top" class="main"><?php echo TEXT_MESSAGE; ?></td>
-                <td><?php echo zen_draw_textarea_field('message', 'soft', '60', '15', htmlspecialchars(empty($_POST['message']) ? strip_tags(TEXT_GV_ANNOUNCE) : stripslashes($_POST['message']), ENT_COMPAT, CHARSET, TRUE), 'class="noEditor"'); ?></td>
-              </tr>
-              <tr>
-                <td colspan="2"><?php echo zen_draw_separator('pixel_trans.gif', '1', '10'); ?></td>
-              </tr>
-              <tr>
-                <td colspan="2" align="right"><?php echo zen_image_submit('button_send_mail.gif', IMAGE_SEND_EMAIL); ?></td>
-              </tr>
-            </table></td>
-          </form></tr>
-<?php
-  }
-?>
-<!-- body_text_eof //-->
-        </table></td>
-      </tr>
-    </table></td>
-  </tr>
-</table>
-<!-- body_eof //-->
-
-<!-- footer //-->
+                <td colspan="3"><?php echo zen_draw_input_field('subject', (!empty($_POST['subject']) ? $_POST['subject'] : ''), 'size="50"'); ?></td>
+            </tr>
+            <tr>
+                <td class="main"><?php echo TEXT_AMOUNT; ?></td>
+                <td colspan="3"><?php echo zen_draw_input_field('amount', (!empty($_POST['amount']) ? $_POST['amount'] : ''), 'step="any" style="padding: 3px 0" required', '', 'number'); ?><?php echo '&nbsp;' . TEXT_AMOUNT_INFO; ?></td>
+            </tr>
+            <tr>
+                <td colspan="4"><hr></td>
+            </tr>
+            <tr>
+                <td colspan="4"><?php echo TEXT_MESSAGE_INFO; ?></td>
+            </tr>
+            <?php if (EMAIL_USE_HTML == 'true') { ?>
+                <tr>
+                    <td class="main" style="vertical-align: top"><?php echo TEXT_HTML_MESSAGE; ?></td>
+                    <td colspan="3"><?php echo zen_draw_textarea_field('message_html', 'soft', '', '10', htmlspecialchars(empty($_POST['message_html']) ? '' : stripslashes($_POST['message_html']), ENT_COMPAT, CHARSET, true),
+                            'id="message_html" class="editorHook"'); ?></td>
+                </tr>
+            <?php } ?>
+            <tr>
+                <td class="main" style="vertical-align: top"><?php echo TEXT_MESSAGE; ?></td>
+                <td colspan="3"><?php echo zen_draw_textarea_field('message', 'soft', '', '10', htmlspecialchars(empty($_POST['message']) ? '' : stripslashes($_POST['message']), ENT_COMPAT, CHARSET, true), 'id="message" class="noEditor tt"'); ?></td>
+            </tr>
+        </table>
+        <div class="form-group">
+            <div class="col-sm-12 text-right">
+                <button type="submit" class="btn btn-primary"><?php echo IMAGE_SUBMIT; ?></button>
+            </div>
+        </div>
+        <?php echo '</form>';
+    }
+    ?>
+    <!-- body_text_eof //-->
+</div>
+<!-- body_eof //--> <!-- footer //-->
 <?php require(DIR_WS_INCLUDES . 'footer.php'); ?>
-<!-- footer_eof //-->
-<br />
+<!-- footer_eof //--> <br/>

```

Comment:
Not sure why this got moved up to be on the same line.
Also, no `/` is needed in `<br>` tags. So, if you're going to reformat this to remove the space, may as well remove the slash.